Legion/DarkheartThicket/ShadeOfXavius.lua

```diff
@@ -36,20 +35,21 @@ end
 -- Event Handlers
 --
 
-function mod:GrowingParanoia(args)
-	self:TargetMessage(args.spellId, args.destName, "Attention", "Alarm")
-	--self:CDBar(args.spellId, 26) -- pull:25.5, 26.8, 28.0, 19.5 / hc pull:27.1, 32.8, 26.7, 27.9 / m pull:25.5, 37.6, 47.3
-end
-
-function mod:GrowingParanoiaApplied(args)
-	self:PrimaryIcon(args.spellId, args.destName)
-	if self:Me(args.destGUID) then
-		self:Say(args.spellId)
+do
+	local function printTarget(self, player, guid)
+		self:TargetMessage(200289, player, "Urgent", "Alert", nil, nil, true)

```

Comment:
Is the change of color and sound for the Message intended?

Reply:
Nope, my bad.